api/src/utils/file-utils.ts

```diff
@@ -25,12 +28,31 @@ const S3 = new AWS.S3({
  */
 export async function getFileFromS3(key: string): Promise<GetObjectOutput> {
   if (!key) {
-    return null;
+    return {};
   }
 
   return S3.getObject({ Bucket: OBJECT_STORE_BUCKET_NAME, Key: key }).promise();
 }
 
+/**
+ * Get file list from S3, based on name prefix
+ * @param prefix
+ */
+export async function getFileListFromS3(prefix: string): Promise<ListObjectsOutput> {
+  if (!prefix) {
+    return {};
```

Comment:
I would return null here instead, and then just change the function response type to `Promise<ListObjectsOutput | null>`

Reply:
Same goes for the getFileFromS3 function above